app/boards/shields/settings_reset/settings_reset.conf

```diff
@@ -1 +1,3 @@
-CONFIG_ZMK_BLE_CLEAR_BONDS_ON_START=y
+CONFIG_ZMK_SETTINGS_RESET_ON_START=y
```

Comment:
So, this would change this shield a bit from what it used to do... I'm not sure that's a bad thing, but probably warrants a blog post or at least announcement on Discord when merged, so folks are aware of the updated functionality.

Reply:
So... the other areas that would be reset by this:

* Output selection
* External power on/off
* Backlight on/off
* RGB on/off and selected effect

Of those... the output selection and external power are maybe the *potentially* most troublesome, but I don't think some docs/info wouldn't help address.

Let's move forward with this.